src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +77,31 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  public boolean getIsGroup() {
+    return isGroup;
+  }
+
+  public void addParticipant(UUID id) {
+    if(isGroup == true){
+      participants.add(id);
+    }
+  }
+
+  public void removeParticipants(UUID id) {
+    if(isGroup == true){
+      participants.remove(id);
+    }
+  }
+
+  public Set<UUID> getParticipants() {
+    if(isGroup == true){
+      return participants;
+    }
+    return new HashSet<UUID>();
```

Comment:
I think you should instead have GroupConversation be a separate class that inherits from Conversation. That way you won't allow methods like this to be called on normal Conversations and you won't need to have the isGroup variable. That'd also be helpful in the future as you might want to add other methods that only GroupConversations can do.